### PR TITLE
WOFF2: An off-curve should stay off even with flags

### PR DESCRIPTION
**Pull request by madig**
This fixes outline corruption of glyphs that start with an off-curve and have e.g. the overlap bit set, previously they'd be turned into on-curves.

### CODE REVIEW

File: Tests/ttLib/woff2_test.py [hunk 1]
```diff
@@ -1220,6 +1221,20 @@ def test_roundtrip_no_glyf_and_loca_tables(self):
 		assert tmp.getvalue() == tmp2.getvalue()
 		assert ttFont.flavor == "woff2"
 
+	def test_roundtrip_overlap_bit(self):
```
Comment (anthrotype) [inline Tests/ttLib/woff2_test.py:1224]: we don't actually "roundtrip" the overlap bit, but we drop it. 
```suggestion
	def test_roundtrip_off_curve_despite_overlap_bit(self):
```

**Review by anthrotype — APPROVED** [review]
🥇

**Comment by madig** [comment]
Thanks for the fix 😬

**Comment by anthrotype** [comment]
thank your for the bug! 🐛

**Comment by justvanrossum** [comment]
Thanks!

(Same bug as #1771, #1772, #1774...)

**Comment by anthrotype** [comment]
it did look familiar indeed :)

**Comment by madig** [comment]
Any other bug we have to fix? :D

**Pull request merged**